library/eclipse-mosquitto

```diff
@@ -1,7 +1,11 @@
 Maintainers: David Audet <david.audet@ca.com> (@daudetCA)
 GitRepo: https://github.com/eclipse/mosquitto.git
 
-Tags: 1.4.10, latest
+Tags: 1.4.12, latest
+GitCommit: 0bb602ed7a89da80d25e6b959a130fdcf0556be5
```

Comment:
Since this commit isn't in `master`, you'll need to also specify a `GitFetch` value (probably `GitFetch: refs/heads/develop`?)